local-cli/default.config.js

```diff
@@ -12,15 +12,7 @@ var path = require('path');
  */
 var config = {
```

Comment:
The changes to this file have been already merged in master, please revert them here :)